src/main/java/com/minecolonies/coremod/colony/buildings/moduleviews/EntityListModuleView.java

```diff
@@ -110,7 +111,7 @@ public void deserialize(@NotNull final PacketBuffer buf)
 
         for (int j = 0; j < size; j++)
         {
-            listOfEntities.add(buf.readResourceLocation());
+            listOfEntities.add(buf.readRegistryIdUnsafe(ForgeRegistries.ENTITIES).getRegistryName());
```

Comment:
You might want to include a null check on this -- it can return null if the ResourceLocation given on the server side wasn't actually in the entities registry (for example, someone saved a mob to the exclude list and then uninstalled the mod that added it).

Ideally you'd check and remove it on the server side as well but it's mostly harmless on that side, and doing it here would also catch weird desyncs.

Reply:
I thought the same thing this morning, removed it on load on the serverside